configure.ac

```diff
@@ -940,7 +940,9 @@ AS_CASE([$cc_basename,$host],
     [oc_ldflags='-brtl -bexpfull'
     AC_DEFINE([HAS_ARCH_CODE32], [1])],
   [gcc*,powerpc-*-linux*],
-    [oc_ldflags="-mbss-plt"])
+    [oc_ldflags="-mbss-plt"],
+  [*,*-*-openbsd*],
```

Comment:
Shouldn't this be just arm64?  The current OCaml trunk works fine for me on OpenBSD-current/amd64.

Reply:
Even the dynlink part? If so, the test can be narrowed to arm64.

Reply:
Sorry, I tell a lie -- I had natdynlink disabled for other reasons in my checkout.  A completely clean trunk build on amd64 failed the natdynlink tests as well,  and the remaining tests passing on amd64 makes sense since that architecture already places literals in .rodata.

I've kicked off a testsuite run with the latest version of your patch on amd64 and arm64 again.